content/dstu2/allergy-intolerance.md

```diff
@@ -84,7 +84,7 @@ To successfully POST an allergy, the following headers must be provided. Allergy
     access-control-max-age → 0
     cache-control → no-cache
     etag → W/"35502191"
-    location → https://fhir.sandboxcernerpowerchart.com/dstu2/9e2aaa6d-3811-4d84-b5f9-93ccf529eefa/AllergyIntolerance/35502191
+    location → https://fhir.sandboxcerner.com/dstu2/9e2aaa6d-3811-4d84-b5f9-93ccf529eefa/AllergyIntolerance/35502191
```

Comment:
fhir.sandboxcerner.com is not a valid URL. Should be fhir-ehr.sandboxcerner.com. This applies to the other closed end-point examples (mostly the writes)